src/components/Button.js

```diff
@@ -21,13 +25,42 @@ const Button = React.createClass({
 
   getDefaultProps () {
     return {
+      isActive: false,
       primaryColor: StyleConstants.Colors.PRIMARY,
       type: 'primary'
     };
   },
 
+  _renderButtonContent () {
+    const styles = this.styles();
```

Comment:
Since your not using the styles object for the else portion of the if statement, I would move this into the the `this.props.isActive` portion.  No need to declare the variable if it won't be used.